### DISCUSSION TITLE
希望添加禁用B站直播P2P以及直播默认选择指定画质的功能

### DISCUSSION THREAD

**Discussion by inventling** [the1812/Bilibili-Evolved#2364] [功能建议 / Ideas]
如题，我今天看直播时才发现B站直播居然占用上传，去问过客服才知道用了P2P，白嫖你的带宽
还有就是B站直播画质不能记住上一次的选择
![QQ截图20211004213352](https://user-images.githubusercontent.com/37429826/135860719-481c186e-cd3f-4f8c-a825-e1b3d1f007b2.png)

**Comment by Geeyun-JY3** [comment]
P2P 重复：#1585
直播默认画质重复：#1324、#1536、#1660
